mlflow/cli.py

```diff
@@ -317,6 +317,9 @@ def _validate_static_prefix(ctx, param, value):  # pylint: disable=unused-argume
     "Default: Within file store, if a file:/ URI is provided. If a sql backend is"
     " used, then this option is required.",
 )
+@click.option(
+    "--artifacts-destination", metavar="URI", default="./artifacts", help="<Add description>",
```

Comment:
How about `mlartifacts` in order to be consistent with `mlruns`?

```suggestion
    "--artifacts-destination", metavar="URI", default="./mlartifacts", help="<Add description>",
```

Reply:
Reminder to add a description :)

Reply:
Thanks, I've updated the default value, I'll add the description soon!